app/src/main/res/navigation/navigation_graph.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/navigation_graph"
+    app:startDestination="@id/splashFragment">
+
+    <fragment
+        android:id="@+id/splashFragment"
+        android:name="com.plenart.organizeme.fragments.SplashFragment"
+        android:label="fragment_splash"
+        tools:layout="@layout/fragment_splash">
+        <action
+            android:id="@+id/action_splashFragment_to_introFragment"
+            app:destination="@id/introFragment"
+            app:popUpTo="@id/splashFragment"
+            app:popUpToInclusive="true" />
+        <action
+            android:id="@+id/action_splashFragment_to_secNavHostFragment"
+            app:destination="@id/secNavHostFragment" />
```

Comment:
If you navigate back from secNavHostFragment, Splash screen is displayed again which is not good. You forgot to add app:popUpTo="@id/splashFragment" and app:popUpToInclusive="true" to remove Splash screen from back stack. You did it right in action_splashFragment_to_introFragment.

Reply:
Done! Forgot to add it back after switching to two navigation graphs. Thanks for heads up!